packages/point-of-sale/src/payments/service.ts

```diff
@@ -9,18 +9,33 @@ import {
 } from '../graphql/generated/graphql'
 import { FnWithDeps } from '../shared/types'
 import { v4 } from 'uuid'
+import { AxiosInstance, AxiosRequestConfig } from 'axios'
 
 type ServiceDependencies = {
   logger: Logger
   config: IAppConfig
   apolloClient: ApolloClient<NormalizedCacheObject>
+  axios: AxiosInstance
 }
 
-type PaymentService = {
+export type WalletAddress = {
+  id: string
+  publicName?: string
+  assetCode: string
+  assetScale: number
+  authServer: string
+  resourceServer: string
+  cardService: string
```

Comment:
Small thing, the `cardService` probably should be optional here since not every wallet address will support card payments. The POS service's payments service thus will need to be able to determine if the wallet address can support payments and fail the request accordingly, so the typing should support that scenario.